COMMUNICATION.md

```diff
@@ -1,58 +1,63 @@
 
-If following hte guidelines don't clear things up consider a call to solve the issue if the matter is pressing, or wait for the next iOS meeting to continue discussing it. '
+If following the guidelines don't clear things up, consider a call to solve the issue if the matter is pressing, or wait for the next iOS meeting to continue discussing it.
 
 ### Good to keep in mind:
-These are the guidelines from GitHub for giving feedback on a PR, there's some valuable points in it for online communication in general:'
-
-> - Familiarize yourself with the context of the issue, and reasons why this Pull Request exists.
-> - If you disagree strongly, consider giving it a few minutes before responding; think before you react.
-> - Ask, don’t tell. (“What do you think about trying…?” rather than “Don’t do…”)
-> - Explain your reasons why code should be changed. (Not in line with the style guide? A personal preference?)
-> - Offer ways to simplify or improve code.
-> - Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced.
-> - Be humble. (“I’m not sure, let’s try…”)
-> - Avoid hyperbole. (“NEVER do…”)
-> - Aim to develop professional skills, group knowledge and product quality, through group critique.
-> - Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Can you use positive language as opposed to neutral?
-> - Use emoji to clarify tone. Compare “✨ ✨ Looks good 👍 ✨ ✨” to “Looks good.”
+These are paraphrased or downright copied over from Github’s pull request guideline:
+
+- Familiarise yourself with the context of the issue.
+- If you disagree strongly, consider giving it a few minutes before responding; think before you react. Try to empathise.
+- Ask, don’t tell. (“What do you think about trying…?” rather than “Don’t do…”)
+- Explain your reasons why code should be changed. (Not in line with the style guide? A personal preference?)
+- Offer ways to simplify or improve code.
+- Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced. Don’t shame, no one started out knowing.
+- Be humble. (“I’m not sure, let’s try…”)
+- NEVER use hyperboles. Unless needed. (“Never store sensitive data in plain text!”)
+- Aim to develop professional skills, group knowledge and product quality, through group critique. That goes both way: try to be open and welcoming to criticism.
+- Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Try to use positive language instead.
+- Use emoji to clarify tone. Compare “well, this sucks” to “well, this sucks 😂.”
```

Comment:
I thought it was a much better example of how an emoji can completely turn around the meaning of a sentence. The first one is super easy to take negatively, the second one is more of a funny remark. I'd probably only use that in a more self-deprecating way. Let's say I made a suggestion and thought it was super good, but everyone else thought it wasn't? Well, this sucks 😂.